build-logic/documentation/src/main/groovy/gradlebuild/docs/ReleaseNotesTransformer.java

```diff
@@ -153,7 +153,7 @@ private void addHighlightJsToHead(Document document) {
 
         head.appendElement("link")
             .attr("rel", "stylesheet")
-            .attr("href", "https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/styles/github.min.css");
+            .attr("href", "https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/styles/default.min.css");
```

Comment:
🤔  I advise against the default theme, because in my view it signals very outdated vibes.

The github theme is must more suitable. We only need to tweat the background color of the code blocks. This should be doable with CSS.

Reply:
Fixed.